types/avs.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+import "github.com/Layr-Labs/eigensdk-go/crypto/bls"
+
+type TaskIndex = uint32
+type TaskDict struct {
+	// We only use a single quorum for incredible squaring
```

Comment:
So we can't use this for multiquroums? This will change once we have that functionality right? Can we add explicitly a TODO to change this later?

Reply:
Good catch. This struct is no longer used. Deleted: [b889aa3](https://github.com/Layr-Labs/eigensdk-go/pull/24/commits/b889aa3b3930741bc4771b9f078871743439e595)